projects/js-packages/licensing/components/activation-screen-error/style.scss

```diff
@@ -0,0 +1,63 @@
+.activation-screen-error__message {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    max-width: 500px;
+    margin-top: calc(var(--spacing-base)*.5);
+
+    svg {
+        margin-left: -3px;
+    }
+
+    span {
+        font-size: 13px;
```

Comment:
I see we have a `--font-body-extra-small` which is 12px and `--font-body-small` which is 14px. My gut tells me we should use one of those just to avoid having too many different font sizes. I understand if design thinks differently though

Reply:
Yeah, my only hesitation here is that I believe Design used this specific font size along with the reduced letter spacing so that all errors remain within one line (on desktop anyway).  If I change the font size up to 14px, a couple of the errors span over 2 lines (with orphan), and if I drop down the size to 12px, then the error seems a bit too small.
I think i'll stick with the Design specified font size in this instance.. Just because of the reason above, and also because (as mentioned in another comment), this change may not remain in the codebase too long due to the upcoming related project.

Reply:
In my experience updating font size just to avoid lines overlapping is not best practices, but like you said this will be changing soon so I think it's fine for now 😅